repository/src/main/java/org/apache/atlas/repository/store/graph/v1/SoftDeleteHandlerV1.java

```diff
@@ -92,7 +93,7 @@ protected void deleteEdge(AtlasEdge edge, boolean force) throws AtlasBaseExcepti
                 AtlasGraphUtilsV2.setEncodedProperty(edge, MODIFIED_BY_KEY, RequestContext.get().getUser());
             }
         }
-        if (isRelationshipEdge(edge))
+        if (isRelationshipEdge)
             AtlasRelationshipStoreV2.recordRelationshipMutation(AtlasRelationshipStoreV2.RelationshipMutation.RELATIONSHIP_SOFT_DELETE, edge, entityRetriever);
```

Comment:
In case of force deletion is there any impact on recordRelationshipMutation method? Since it is using the same edge.